src/Http/Controllers/Api/BaseController.php

```diff
@@ -27,7 +27,7 @@ protected function bulkActionResponse(int $rowsAffected, string $transKey): Resp
         return new Response([
             'success' => true,
             'rows_affected' => $rowsAffected,
-            'message' => trans_choice("forum::{$transKey}", $rowsAffected),
+            'message' => trans_choice("forum.{$transKey}", $rowsAffected),
```

Comment:
Just for my own information - what's the reason for this change? Is it due to a change in how Laravel handles paths?

Reply:
That change was accidental. Has been fixed.